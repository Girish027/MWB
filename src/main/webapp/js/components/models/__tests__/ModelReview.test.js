import React from 'react';
import { mount, shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router';
import store from 'state/configureStore';

const ModelReview = require('components/models/ModelReview').default;

let wrapper;

describe('<ModelReview />', () => {
  describe('Creating an instance', () => {
    beforeAll(() => {
      wrapper = mount(<Provider store={store}>
        <MemoryRouter
          initialEntries={['/projects/-1/models/createconfig']}
          initialIndex={0}
        >
          <ModelReview />
        </MemoryRouter>
      </Provider>);
    });

    test('should exist', () => {
      expect(wrapper.exists()).toBe(true);
    });
  });

  describe('Snapshots', () => {
    beforeEach(() => {
      wrapper = shallow(<ModelReview />);
    });

    afterEach(() => {
      jest.clearAllMocks();
    });
    test('should match with basic props', () => {
      expect(toJSON(wrapper)).toMatchSnapshot();
    });
  });
});
