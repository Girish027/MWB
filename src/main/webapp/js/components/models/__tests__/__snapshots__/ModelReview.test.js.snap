// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModelReview /> Snapshots should match with basic props 1`] = `""`;
